### PR TITLE
feat: trace presentation

### PR DESCRIPTION
**Pull request by nissimsan**
Brought over from https://github.com/w3c-ccg/traceability-interop/pull/70

### CODE REVIEW

File: docs/openapi/components/schemas/common/TraceablePresentation.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        "https://w3id.org/security/suites/jws-2020/v1",
+        "https://w3id.org/traceability/v1"
+      ],
+    "id": "urn:uuid:83432751123654",
+    "type": 
+      [
+        "VerifiablePresentation",
+        "TraceablePresentation"
+      ],
+    "workflow": {
```
Comment (mkhraisha) [inline docs/openapi/components/schemas/common/TraceablePresentation.yml:25]: This uses the word workflow, in our definition we have a path defined not a workflow, really need to settle on one.

also the instance field, is this supposed to be correlationID?
what is definition field?

File: docs/openapi/components/schemas/common/Workflow.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+title: Workflow
```
Comment (mkhraisha) [inline docs/openapi/components/schemas/common/Workflow.yml:1]: This uses the word workflow, in our definition we have a path defined not a workflow, really need to settle on one.

also the instance field, isn't this supposed to be a correlationId?

is the intention of the definition field to link to a specific type of workflow? something like "import avocadoes to US"? or is it supposed to be something more akin to "company A imports avocadoes via port X to Country C"? 
in the example above you have a uuid what does that reference/represent?

**Review by mprorock — APPROVED** [review]

File: docs/openapi/components/schemas/common/TraceablePresentation.yml [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+      "instance": [
+        "f5fb6ce4-b0b1-41b8-89b0-331ni58b7ee0"
+      ],
+      "definition": [
+        "n1552885-cc91-4bb3-91f1-5466a0be084e"
```
Comment (OR13) [inline docs/openapi/components/schemas/common/TraceablePresentation.yml:30]: used to resolve a process diagram.

File: docs/openapi/components/schemas/common/TraceablePresentation.yml [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+      ],
+    "id": "urn:uuid:83432751123654",
+    "type": 
+      [
+        "VerifiablePresentation",
+        "TraceablePresentation"
+      ],
+    "workflow": {
+      "instance": [
+        "f5fb6ce4-b0b1-41b8-89b0-331ni58b7ee0"
```
Comment (OR13) [inline docs/openapi/components/schemas/common/TraceablePresentation.yml:27]: used to group instances of a process that was executed.

File: docs/openapi/components/schemas/common/Workflow.yml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+type: object
+description: A workflow instance and definition. 
+  - type: object
+    properties:
+      definition: 
+        description: Identifier of a particular type of workflow.
```
Comment (OR13) [inline docs/openapi/components/schemas/common/Workflow.yml:7]: refer to correlation id?

**Review by OR13 — APPROVED** [review]

**Review by mkhraisha — APPROVED** [review]

**Comment by OR13** [comment]
See related issue on interop: https://github.com/w3c-ccg/traceability-interop/issues/85

**Pull request merged**